flakey-test-tracking/src/config.js

```diff
@@ -0,0 +1,32 @@
+// shouldTrackFlakes => tests are retried `numRetries` times and flakey results are logged w/ test output
+const shouldTrackFlakes =
+  (process.env.CIRCLECI && process.env.FLAKEY !== 'false') || process.env.FLAKEY === 'true'
```

Comment:
So I've set it up this way so that the default when running locally is to not use the flake tracker (you have to set FLAKEY = true to use it) and the default when running in CI is to use the flake tracker (you have to set FLAKEY=false to not use it)

Reply:
so, empty in circle-ci means true, and in the other environments, false. I think that it could be misleading at the beginning. Try to document that somewhere, haha

Reply:
Right, I've added it to the README and left a comment in the config file